docker-images/stunnel-zookeeper/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+
+RUN yum -y install stunnel && yum clean all -y
+
+# set Stunnel home folder
+ENV STUNNEL_HOME=/opt/stunnel
+
+# exposing Zookeeper ports
+EXPOSE 2888 3888
+
+RUN mkdir $STUNNEL_HOME && mkdir -p -m g+rw /usr/local/var/run/
+
+# copy scripts for starting Stunnel
+COPY ./scripts/ $STUNNEL_HOME
```

Comment:
If you do not want to have a base image ... shouldn't the `COPY` command be the last one (i.e. between `WORKDIR` and `CMD`) to make better use of the layering?

Reply:
Should we do the same for `kafka-base` ?

Reply:
The `kafka-base` is used as a base image. So I do not think changing it there could improve how the layers are shared and reused by other images. TBH I'm actually not 100% sure whether this here impacts the layering and reusing of the layers. So feel free to ignore this if you want.

Reply:
I can do the change, I was just wondering if the impact was the same in the `kafka-base` image.

Reply:
Yeah ... I'm sure this is not a problem there because its a base image. Here we don't have the base image but will have multiple stunnel images ... so the sharing through layers might be useful.

Reply:
Maybe it's better developing a base image in this PR for having it ready for the future tls sidecars on other pods.

Reply:
BTW: One more think I'm missing here ... in the `kafka-base` we add the user to not run it under `root` or whatever the default in CentOS is ... I do not see that here.